docs/useful_tools.md

```diff
@@ -345,3 +345,20 @@ plt.show()
 You should see something similar to:
 
 ![3dogs_mask](../resources/3dogs_mask.png)
+
+And you can use `test_torchserver.py` to compare result of torchserver and pytorch, and visualize them.
+
+```shell
+python tools/test_torchserver.py ${IMAGE_FILE} ${CONFIG_FILE} ${CHECKPOINT_FILE} ${MODEL_NAME}
+[--inference-addr ${INFERENCE_ADDR}] [--result-image ${RESULT_IMAGE}] [--device ${DEVICE}]
+```
+
+Example:
+
+```shell
+python tools/deployment/test_torchserver.py \
```

Comment:
tools/deployment/test_torchserver.py --> tools/test_torchserver.py
or move tools/test_torchserver.py to tools/deployment/test_torchserver.py.

Reply:
Sorry,I have fix it in my local but I forget push it. (-_-)||    I will push it later.